lckernel/cad/math/intersectionhandler.cpp

```diff
@@ -289,3 +289,66 @@ void Intersection::BezCir(const geo::Circle C, const geo::Area c_area, const geo
     BezCir(C, c_area, m_area, b2split[1], ret);
 
 }
+
+std::vector<geo::Coordinate> Intersection::SplineLine(geo::Spline B, const geo::Vector& V) {
+    std::vector<geo::Coordinate> ret;
+    auto beziers = B.beziers();
+    for(const auto & bezier : beziers) {
+        auto vecret = BezierLine(bezier, V);
```

Comment:
Can we have all method functions start with a lowercase letter?
Like this : auto vecret = bezierLine(bezier, V); instead of this auto vecret = BezierLine(bezier, V);

All class names should start with a Uppercase name, all method names should start with a lowercase name.

For example this code : auto vecret = BezierLine(bezier, V);
Looks a lot like if as object is constructed, this feel get's enforced by the fact it's auto..

When it looks like this: auto vecret = bezierLine(bezier, V); it feels like it's a method call.

Even better would be the following naming : auto vecret = getBezierLineIntersections(bezier, V); which make's it very clear what's happening.